controls/radchartview/features/plot-area.md

```diff
@@ -51,6 +51,42 @@ You can customize the plot area visual via the __PlotAreaStyle__ property of the
 #### __Figure 3: PlotAreaStyle example__
 ![PlotAreaStyle example](images/radchartview-features-plot-area-2.png)
 
+## Displaying stacked elements 
```

Comment:
The title of the section is a bit misleading. What are stacked elements? Are they invisible by default? 

Use something more familiar for the users. For example "Changing ZIndex" or "Controlling the stack order of the elements". Usually, most developers knows what ZIndex is and it is highly possible that they will start searching for this word.